# Goal - Either a set of target joints OR a target point.
#        (If both are given, the joints will take precedent and the point will be ignored.)
JointsCommand target_joints     # [head pan, head tilt]
PointCommand target_point
---
# Result - Success result & final head position
bool success
float64[] head_position         # [head pan, head tilt]
---
# Feedback - Current head position
float64[] head_position         # [head pan, head tilt]
